docs/source/tutorials/token_classification.ipynb

```diff
@@ -0,0 +1,757 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4f4ebdc0",
+   "metadata": {},
+   "source": [
+    "<details><summary>Below is the code used to read the `.npz` file. </summary> \n",
+    "    \n",
+    "    def read_npz(filepath): \n",
+    "        data = dict(np.load(filepath)) \n",
+    "        data = [data[str(i)] for i in range(len(data))] \n",
+    "        return data </details> "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "ab9d59a0",
+   "metadata": {
+    "nbsphinx": "hidden"
+   },
+   "outputs": [],
+   "source": [
+    "# TODO: hide me \n",
+    "def read_npz(filepath): \n",
+    "    data = dict(np.load(filepath)) \n",
+    "    data = [data[str(i)] for i in range(len(data))] \n",
+    "    return data "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "519cb80c",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "File ‘pred_probs.npz’ already there; not retrieving.\r\n",
+      "\r\n"
+     ]
+    }
+   ],
+   "source": [
+    "!wget -nc 'https://cleanlab-public.s3.amazonaws.com/TokenClassification/pred_probs.npz' \n",
+    "\n",
+    "pred_probs = read_npz('pred_probs.npz') "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1dc3150f",
+   "metadata": {},
+   "source": [
+    "## 3. Use cleanlab to find potential label issues \n",
+    "\n",
+    "Based on the given labels and out-of-sample predicted probabilities, cleanlab can quickly help us identify label issues in our dataset. Here we request that the indices of the identified label issues be sorted by cleanlab’s self-confidence score, which measures the quality of each given label via the probability assigned to it in our model’s prediction. Given that we would like to visualize the results, we first read the files. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1e6e2963",
+   "metadata": {},
+   "source": [
+    "Let's look at the top 20 examples cleanlab thinks are most likely to be incorrectly labeled. We obtain the sentences from the original files to display the word-level token label issues in context. `given_words` contains the word-level tokens in the dataset such that `given_words[i]` is a list of words of the `i`'th sentence; `given_labels` contains the given labels in the dataset such that `given_labels[i]` is a list of labels of the `i`th sentence. Note that in CoNLL-2003, the `B-` and `I-` prefixes indicates whether the tokens are at the start of an entity, which are ignored in this tutorial. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ac24b72b",
+   "metadata": {},
+   "source": [
+    "<details><summary>Below is the code used for reading the files.</summary>\n",
+    "\n",
+    "    filepaths = ['data/train.txt', 'data/valid.txt', 'data/test.txt'] \n",
+    "    entities = ['O', 'B-MISC', 'I-MISC', 'B-PER', 'I-PER', 'B-ORG', 'I-ORG', 'B-LOC', 'I-LOC']\n",
```

Comment:
Can you give this variable a different name since it does not correspond to the classes in our token classification problem, but that variable has the same nam. It would also be good to clarify what this variable is for, and to clarify the classes in our token classification problem are slightly different (merged) version of these.